docker/ol-web-start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+python --version
+authbind --deep \
+  scripts/openlibrary-server conf/openlibrary.yml \
+  --gunicorn \
+  --reload \
+  --workers 4 \
```

Comment:
We should consider upgrading and tailoring our gunicorn setup to use sane settings for workers + threads
https://docs.gunicorn.org/en/stable/settings.html

See: https://twitter.com/noufalibrahim/status/1297030284617986048 (gunicorn chat w/ 2 previous OL devs)